apps/labs/posts/code-generator-and-nebari.md

```diff
@@ -0,0 +1,159 @@
+
+This app was created with the great efforts of members of the PyTorch-Ignite community, particularly @[ydcjeff](https://github.com/ydcjeff) @[trsvchn](https://github.com/trsvchn) @[vfdev-5](https://github.com/vfdev-5).
+
+![The Code Generator App](/posts/code-generator-and-nebari/app.png)
+
+# The Issues I worked on
+
+My tasks were primarily to work on features that could improve the reproducibility-related features of the templates. Also, I had to contribute to new features and libraries which can be best for experiments configuration management. My contributions to the project were mainly three-fold:
+
+### 1. Working on enhancing templates and existing reproducibility scope of the results from experiments
+
+In this part, I worked on improving the readability of the templates and improving the API of the templates. This led me to make some necessary API changes and test the existing libraries used for writing the research literature in data science and Kaggle competitions. One funny error we caught in the PyYaml project was
+
+```python
+#~~ Saving config.yaml for reproducibility
+#-------------------------------------
+## Config file 1 (config1.yaml)
+lr: 1e-10
+
+## Config file 2 (config2.yaml)
+lr: 1.0e-10
+#-------------------------------------
+
+with open("config1.yaml", "r") as f1:
+	config1 = yaml.safe_load(f1) # loads lr -> string
+>> config1 = { 'lr': '1e-10' }
+
+with open("config2.yaml", "r") as f2:
+  config2 = yaml.safe_load(f2) # loads lr -> python-floating point
+>> config2 = { 'lr': 1e-10 } # (the expected behaviour)
+```
+
+There were other problems as well, like not being able to store Python objects using `PyYAML API`. This led us to adopt [OmegaConf.DictConfig](https://omegaconf.readthedocs.io/en/2.3_branch/api_reference.html#the-omegaconf-api) as the API standard for maintaining config data in the templates. This worked great on testing and is well maintained by the developers. Also, methods like resolution of config dict are very well written in this `OmegaConf` project.
+
+Also, we added support for [Hydra](http://hydra.cc) `and` [Python-Fire](https://github.com/google/python-fire/blob/master/docs/guide.md) as they are great in terms of configuration management and also worked very well with PyTorch-Ignite. They also allow overriding existing configurations using the following bash commands.
```

Comment:
Is there a reason why you are typing `and` with a monospace font here? If not I'd recommend removing the extra formatting 🙂